test-suite/misc/coqtop_print-mod-uid.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+export COQBIN=$BIN
+export PATH=$COQBIN:$PATH
+
+TMP=`mktemp -d`
+cd $TMP
```

Comment:
We usually put the files in a misc/ subdirectory.
For this test I would just use prerequisite/admit instead of making new files though.